src/components/MultiStepIndicator/__stories__/helper.ts

```diff
@@ -1,6 +1,6 @@
-import MultiStepIndicator from "../MultiStepIndicator";
+import MultiStepIndicator, { type Step } from "../MultiStepIndicator";
```

Comment:
Maybe worth moving `Step` to constants

Reply:
Thanks for the review @SergeyRoyt :)
@hadasfa and I discussed about it and decided that since it is coupled to MultiStepIndicator and it's relevant usages, it might be better to have this connection between them, since Step should never be used unless it's for MultiStepIndicator implementation

I'm not sure actually about industry standards best practice for it
I'll make further check

Reply:
From what I've checked, in case Step is always used only when MultiStep is being imported and used, it makes more sense to export it directly from the component, it seems to be a pretty wide standard (which we use on other components also)
is that ok to leave it that way, @SergeyRoyt?

Reply:
Yeah, I think it's ok